monero/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1.1
+id: monero
+category: Finance
+name: Monero Node
+icon: https://raw.githubusercontent.com/deverickapollo/umbrel-monero/dev/ui/src/assets/community-monero-icon.png
+version: "0.18.2.2"
+tagline: Run your monero node and independently store and validate every single Monero transaction with it.
+description: >-
+  Run your monero node and independently store and validate every single Monero transaction with it.
+  This is a full node that will download the entire Monero blockchain and store it on your Umbrel. 
+  This is the most secure way to run a Monero node, but it will take a long time to sync and will require a lot of storage space.
+developer: Monero
+website: https://www.getmonero.org/
+dependencies: []
+repo: https://github.com/deverickapollo/umbrel-monero
+support: https://github.com/deverickapollo/umbrel-monero/issues
+port: 9976
+submitter: Deverick
+submission: https://github.com/getumbrel/umbrel-apps/pull/690
+gallery:
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Monero%20Dashboard%20Menu.png
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Connect%20Wallet.png
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Advanced%20Settings.png
+path: ""
+defaultPassword: ""
+releaseNotes: >-
+ Initial Release
```

Comment:
For this initial release we can just leave this as 
```
releaseNotes: ""
```

and then they won't show in the UI, but the yaml will remain to remind us to update it in future app versions.